Core2/Onboarding/build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.compose.compiler)
+}
+
+val sharedCompileSdk: Int by rootProject.extra
+val sharedMinSdk: Int by rootProject.extra
+val sharedJavaVersion: JavaVersion by rootProject.extra
+
+android {
+    namespace = "com.infomaniak.library.onboarding"
+    compileSdk = sharedCompileSdk
+
+    defaultConfig {
+        minSdk = sharedMinSdk
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+    compileOptions {
+        sourceCompatibility = sharedJavaVersion
+        targetCompatibility = sharedJavaVersion
+    }
+    buildFeatures {
+        compose = true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.5.14"
+    }
+    kotlinOptions {
+        jvmTarget = sharedJavaVersion.toString()
+    }
+}
+
+dependencies {
+
+    implementation(core2.androidx.core.ktx)
```

Comment:
```suggestion
    implementation(libs.androidx.core.ktx)
```

Reply:
You resolved the conversation but didn't update the code.

Reply:
Not sure how I understood your comment the first time because I added a "thumbs up" but actually we don't want to depend on the SwissTransfer module inside of Core2